### ISSUE TITLE
Add a field in the questionnaire from a database like the address of the replier enhancement

### ISSUE THREAD

**Issue by julienfr112** [sdaps/sdaps#11]
I'm trying to modify the latex.py in the stamp directory to modify the tex text , for example remplacing Address by the value of a dict that i would have populate maybe in the ids command.

**Comment by tornadof3** [comment]
Hello

Thank you for the proof of concept about adding eg 'name of student' to top of test paper based on ID etc.

I can't get the stamp command as above to work with the suggested commands and sample data. Here's the output below. Have there been any recent changes to SDAPS that prevent this working? I'm new to SDAPS today and using v1.1.8 [it seems great!]. Thanks.

user@user-VirtualBox:~/quest1/quest2$ sdaps test stamp --existing
------------------------------------------------------------------------------
- SDAPS -- stamp
------------------------------------------------------------------------------
Running pdflatex now twice to generate the stamped questionnaire.
This is pdfTeX, Version 3.14159265-2.6-1.40.15 (TeX Live 2014/Debian) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
This is pdfTeX, Version 3.14159265-2.6-1.40.15 (TeX Live 2014/Debian) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
Error: Error running "pdflatex" to compile the LaTeX file.
Error: An error occured during creation of the report. Temporary files left in '/tmp/tmpqv2ZdN'.
Traceback (most recent call last):
  File "/usr/bin/sdaps", line 23, in <module>
    sys.exit(sdaps.main())
  File "/usr/lib/python2.7/dist-packages/sdaps/__init__.py", line 64, in main
    return cmdline['_func'](cmdline)
  File "/usr/lib/python2.7/dist-packages/sdaps/script.py", line 105, in decorated_function
    result = function(cmdline)
  File "/usr/lib/python2.7/dist-packages/sdaps/cmdline/stamp.py", line 56, in stamp
    return stamp.stamp(survey, output, cmdline)
  File "/usr/lib/python2.7/dist-packages/sdaps/stamp/__init__.py", line 90, in stamp
    create_stamp_pdf(survey, output_filename, questionnaire_ids)
  File "/usr/lib/python2.7/dist-packages/sdaps/stamp/latex.py", line 67, in create_stamp_pdf
    raise AssertionError('PDF file not generated')
AssertionError: PDF file not generated

**Comment by benzea** [comment]
The error message is not very helpful as is. Could you provide the generated .log file (it is in the temporary folder '/tmp/tmpqv2ZdN' in this case). It might be a lot of different things, but i don't think that the general concept failed. It is more likely that the ID is simply not in the file containing the data is not in the SDAPS folder or something.

**Comment by tornadof3** [comment]
Ah, so does this mean it won't work at present?

**Comment by benzea** [comment]
Yes. I think it was broken by commit a6ca773ef18995548da3f7715e9cfa1c41b3138a so 1.1.7 is not affected by this particular bug, but 1.1.8 is. I have not yet investigated it more.

**Comment by benzea** [comment]
Well, that was an easy fix.

**Comment by tornadof3** [comment]
OK, that last commit seems to have worked well, thanks
